src/prefect/runner/submit.py

```diff
@@ -0,0 +1,196 @@
+)
+from prefect.states import Pending
+from prefect.tasks import Task
+from prefect.utilities.asyncutils import sync_compatible
+
+logger = get_logger("webserver")
+
+
+async def get_current_run_count() -> int:
+    async with httpx.AsyncClient() as client:
+        response = await client.get(
+            f"http://{PREFECT_RUNNER_SERVER_HOST.value()}"
+            f":{PREFECT_RUNNER_SERVER_PORT.value()}/run_count"
+        )
+        response.raise_for_status()
+        return response.json()
+
+
+async def _submit_flow_to_runner(
+    flow: Flow,
+    parameters: Dict[str, Any],
+    # capture_errors: bool = SETTING.value()?
+) -> uuid.UUID:
+    """
+    Run a callable in the background via the runner webserver.
+
+    Args:
+        prefect_callable: the callable to run, e.g. a flow or task
+        parameters: the keyword arguments to pass to the callable
+        timeout: the maximum time to wait for the callable to finish
+        poll_interval: the interval (in seconds) to wait between polling the callable
+    """
+    from prefect.engine import (
+        _dynamic_key_for_task_run,
+        collect_task_run_inputs,
+        resolve_inputs,
+    )
+
+    async with get_client() as client:
+        parent_flow_run_context = FlowRunContext.get()
+
+        task_inputs = {
+            k: await collect_task_run_inputs(v) for k, v in parameters.items()
+        }
+
+        dummy_task = Task(name=flow.name, fn=flow.fn, version=flow.version)
+        parent_task_run = await client.create_task_run(
+            task=dummy_task,
+            flow_run_id=parent_flow_run_context.flow_run.id,
+            dynamic_key=_dynamic_key_for_task_run(parent_flow_run_context, dummy_task),
+            task_inputs=task_inputs,
+            state=Pending(),
+        )
+
+        parameters = await resolve_inputs(parameters)
+
+        response = await client._client.post(
+            (
+                f"http://{PREFECT_RUNNER_SERVER_HOST.value()}"
+                f":{PREFECT_RUNNER_SERVER_PORT.value()}"
+                "/flow/run"
+            ),
+            json={
+                "entrypoint": flow._entrypoint,
+                "parameters": flow.serialize_parameters(parameters),
+                "parent_task_run_id": str(parent_task_run.id),
+            },
+        )
+        response.raise_for_status()
+
+        flow_run_id = response.json()["flow_run_id"]
+
+        return uuid.UUID(flow_run_id)
+
+
+@sync_compatible
+async def submit_to_runner(
+    prefect_callable: Union[Flow, Task],
+    parameters: Dict[str, Any],
+) -> uuid.UUID:
+    """
+    Run a callable in the background via the runner webserver.
+
+    Args:
+        prefect_callable: the callable to run (only flows are supported for now, but eventually tasks)
+        parameters: the keyword arguments to pass to the callable,
+    """
+
+    flow_run_id = await _submit_flow_to_runner(prefect_callable, parameters)
+
+    if inspect.isawaitable(flow_run_id):
```

Comment:
It looks like `flow_run_id` will always be a UUID, am I reading that wrong?

Reply:
do you mean as opposed to a coro or a string?

but either way i've been (non-rigorously) assuming always UUID, happy to tighten that up as needed